src/Orleans.Core/Messaging/Message.cs

```diff
@@ -548,8 +549,9 @@ public static Message CreatePromptExceptionResponse(Message request, Exception e
             };
         }
 
-        internal void DropExpiredMessage(MessagingStatisticsGroup.Phase phase)
+        internal void DropExpiredMessage(ILogger logger, MessagingStatisticsGroup.Phase phase)
         {
+            logger.LogWarning((int)ErrorCode.Messaging_DroppingExpiredMessage, "Dropped expired message during {Phase} phase.  Message: {Message}", Enum.GetName(typeof(MessagingStatisticsGroup.Phase), phase), logger.IsEnabled(LogLevel.Trace) ? this.ToLongString() : this.ToString());
```

Comment:
Can't we just use `phase` here without the (expensive) `Enum.GetName` call?
![image](https://user-images.githubusercontent.com/203839/67049518-3fcea980-f0eb-11e9-8d86-12d786094b67.png)

Other than that, there's a double-space in the message.